coverage/lcov-report/base.css

```
@@ -0,0 +1,362 @@
+body,
+html {
+    margin: 0;
+    padding: 0;
+    height: 100%;
+}
+body {
+    font-family:
+        Helvetica Neue,
+        Helvetica,
```

Biome (1.9.4)

[error] Duplicate font names are redundant and unnecessary: Helvetica

Remove duplicate font names within the property

(lint/suspicious/noDuplicateFontNames)